### DISCUSSION TITLE
Yo, I'm interested in this

### DISCUSSION THREAD

**Discussion by ghost** [RiversideValley/Emerald#41] [General]
Is there any roadmap for this project? If so, I would like to know what's in the pipeline and what I can do to improve this project.

**Comment by ghost** [comment]
As I understand it, current goal looks like to be porting Emerald to another platform. It shouldn't be a problem as Uno Platform is a thing but lemme take a look at the app's code xD.

**Reply by Lamparter** [reply]
Yes, the team are interested in porting Emerald to another platform via Uno Platform or another appropriate alternative.

**Reply by NoobNotFound** [reply]
if you know something better than Uno that really feels like "native", feel free to say it here.
